### PR TITLE
Add ESLint Plugin Package

### PR DESCRIPTION
**Pull request by anthlasserre**

### CODE REVIEW

File: packages/eslint-plugin-react-native/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Running unit tests
+
+Run `nx test eslint-plugin-react-native` to execute the unit tests via [Jest](https://jestjs.io).
+
+# @getluko/eslint-plugin-mobile-guidelines
+
+## Installation
+
+```
+yarn add @getluko/eslint-plugin-mobile-guidelines --dev
```
Comment (kimak) [inline packages/eslint-plugin-react-native/README.md:18]:
```suggestion
yarn add @getluko/eslint-plugin-react-native --dev
```

File: packages/eslint-plugin-react-native/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+```
+
+## Configuration
+
+`.eslintrc.js`
+
+```
+module.exports = {
+  plugins: ['@getluko/mobile-guidelines'],
```
Comment (kimak) [inline packages/eslint-plugin-react-native/README.md:27]:
```suggestion
  plugins: ['@getluko/eslint-plugin-react-native],
```

File: packages/eslint-plugin-react-native/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  rules: {
+    '@getluko/mobile-guidelines/lower-dash-case-test-id': 1,
```
Comment (kimak) [inline packages/eslint-plugin-react-native/README.md:29]:
```suggestion
    '@getluko/react-native/lower-dash-case-test-id': 1,
```

File: packages/eslint-plugin-react-native/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  }
+```
+
+## Rules
+
+### [@getluko/mobile-guidelines/lower-dash-case-test-id](src/docs/rules/lower-dash-case-test-id.md)
```
Comment (kimak) [inline packages/eslint-plugin-react-native/README.md:35]:
```suggestion
### [@getluko/react-native/lower-dash-case-test-id](src/docs/rules/lower-dash-case-test-id.md)
```

Comment (kimak) [inline packages/eslint-plugin-react-native/README.md:35]: Should we keep the documentation at the root for now? We will not have 100 to maintain so it should be easier

File: packages/eslint-plugin-react-native/src/docs/rules/lower-dash-case-test-id.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+Allows you to enforce a consistent naming pattern for testID prop which expect a lower dash case.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```jsx
+return (
+  <Element
+    testID={'ElementTestID'}
```
Comment (kimak) [inline packages/eslint-plugin-react-native/src/docs/rules/lower-dash-case-test-id.md:12]:
```suggestion
    testID="ElementTestID"
```
Do we need bracket?

**Review by kimak — CHANGES_REQUESTED** [review]

File: packages/eslint-plugin-react-native/README.md (hunk 2 above)
Comment (anthlasserre) [reply packages/eslint-plugin-react-native/README.md:27]: We can remove `eslint-plugin` at the beginning of the name. ESLint is translating it automatically.

`@getluko/react-native`

File: packages/eslint-plugin-react-native/src/docs/rules/lower-dash-case-test-id.md (hunk 5 above)
Comment (anthlasserre) [reply packages/eslint-plugin-react-native/src/docs/rules/lower-dash-case-test-id.md:12]: No indeed 👍🏼

File: packages/eslint-plugin-react-native/README.md (hunk 4 above)
Comment (anthlasserre) [reply packages/eslint-plugin-react-native/README.md]: I added a preview also at the root README. But I think it's cleaner to keep the rule markdown file as well to well explain how configure the rule.

File: README.md [hunk 6]
```diff
@@ -12,18 +12,22 @@
...
   ```jsx
     // bad
     return (
       <Element
-        testID={'ElementTestID'}
+        testID="ElementTestID"
       />
     );
 
     // good
```
Comment (kimak) [inline README.md:27]:
```suggestion
    // good ✅
```
Emoji always make it clearer ^^

**Review by kimak — APPROVED** [review]

**Pull request merged**